### DISCUSSION TITLE
Network Deployment, need to know how to make iso

### DISCUSSION THREAD

**Discussion by talibsyed** [meetrevision/playbook#109] [Q&A] [answered]
I need to deploy revi.cc on 14 desktop computers in a lab. For this purpose, I need to make an ISO just like revi used to provide, but I don't know how to make one. Can someone list the steps to carry to finally make an ISO with revi.cc updates already applied or automatically applied during installation or during OOBE.

**Comment by melo936** [comment]
Unfortunately, it is currently not possible to make an ISO from a playbook, because it relies on AME Wizard. To automate, I guess it's possible to integrate trusted-installer-cli and a playbook into an ISO and run it after OOBE. However, I don't know how well it works without AME Wizard GUI

**Comment by melo936** [comment, marked as answer]
ISO Injection is now available. Closing this.
